src/storage/src/hummock/state_store.rs

```diff
@@ -452,7 +490,21 @@ impl StateStore for HummockStorage {
         R: RangeBounds<B> + Send,
         B: AsRef<[u8]> + Send,
     {
-        self.iter_inner::<R, B, ForwardIter>(key_range, read_options)
+        self.iter_inner::<_, _, ForwardIter>(None, key_range, read_options)
+    }
+
+    fn prefix_iter<R, B>(
+        &self,
+        _prefix_hint: Option<Vec<u8>>,
+        key_range: R,
+        read_options: ReadOptions,
+    ) -> Self::PrefixIterFuture<'_, R, B>
+    where
+        R: RangeBounds<B> + Send,
+        B: AsRef<[u8]> + Send,
+    {
+        // TODO: transfer prefix_hint to iter_inner next pr
```

Comment:
why not just using it in this PR? will there be further changes in the interfaces in the next PR?

Reply:
In the context of this PR, sstable does not contain the prefix in bloom_filter , if we transfer the `prefix_hint `, `iter_inner` will drop all the sstable and makes data query errors.

So I will introduce in the next pr